packages/typescript-estree/src/create-program/createProjectProgram.ts

```diff
@@ -52,10 +52,10 @@ function createProjectProgram(
   );
   const describedPrograms =
     relativeProjects.length === 1
-      ? relativeProjects[0]
+      ? ` ${relativeProjects[0]}`
       : `\n${relativeProjects.map(project => `- ${project}`).join('\n')}`;
   const errorLines = [
-    `ESLint was configured to run on \`${describedFilePath}\` using \`parserOptions.project\`: ${describedPrograms}`,
+    `ESLint was configured to run on \`${describedFilePath}\` using \`parserOptions.project\`:${describedPrograms}`,
```

Comment:
Not strictly needed for this PR, but it was _so annoying_ having Prettier auto-save the snapshot to no longer have a trailing slash...